packages/dd-trace/src/opentracing/propagation/text_map.js

```diff
@@ -325,6 +325,9 @@ class TextMapPropagator {
       if (context === null) {
         context = extractedContext
         if (this._config.tracePropagationExtractFirst) {
+          if (this._hasPropagationStyle('extract', 'baggage') && carrier.baggage) {
```

Comment:
I just noticed that this condition could also be moved to `_extractBaggageItems`.